crates/rolldown/tests/esbuild/loader/loader_data_url_base64_invalid_utf8/artifacts.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/loader/loader_data_url_base64_invalid_utf8
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+//#region binary.txt
+var binary_default = "data:text/plain;charset=utf-8,ÿ";
```

Comment:
Esbuild:

```js
var binary_default = "data:text/plain;charset=utf-8;base64,/w==";
```

It's because in default, `esbuild` parses the non-ascii chars. Result here:

```bash
(base) ethangoh@Ethans-MacBook-Pro loader_data_url_base64_invalid_utf8 % esbuild entry.js --bundle --loader:.txt=dataurl --charset=utf8
(() => {
  // binary.txt
  var binary_default = "data:text/plain;charset=utf-8,ÿ";

  // entry.js
  console.log(binary_default);
})();
```

Reply:
It turns out that in `go`, the `\xFF` is not the same as in JavaScript or Python. I'll use go's immediately.